web/components/notifications/notification-dropdown.tsx

```diff
@@ -73,15 +77,46 @@ function useNotificationPreferenceItem(notification: Notification) {
         ) : (
           <PlusCircleIcon className="h-5 w-5" />
         ),
-        onClick: () => {
+        onClick: () =>
           changeSetting({
             setting: 'browser',
             newValue: !inAppEnabled,
             privateUser: privateUser,
             subscriptionTypeKey: subType,
-          })
-        },
-      },
+          }),
+      } as DropdownItem,
     ]
   } else return []
 }
+
+function useNotificationFollowItem(notification: Notification) {
+  const sourceContractId = notification.sourceContractId
+  const sourceContractSlug = notification.sourceContractSlug
+  const followers = useContractFollows(sourceContractId)
+  const user = useUser()
+  if (!user || !sourceContractId || !sourceContractSlug || !followers) {
+    return []
+  }
+  const isFollowing = followers?.includes(user.id)
```

Comment:
`followers` is confusing if it refers to the contractIds you are watching.

Wait, I think `isFollowing` should be checking if your contract follows includes `sourceContractId`, not `user.id`

Reply:
is that right? I thought followers is the list of people following a contract, so you're looking for yourself in that list

Reply:
yee it's how FollowMarketButton uses it
```
export const FollowMarketButton = (props: {
  contract: Contract
  user: User | undefined | null
}) => {
  const { contract, user } = props
  const followers = useContractFollows(contract.id)
  const [open, setOpen] = useState(false)

  const watching = followers?.includes(user?.id ?? 'nope')
```

Reply:
Oh, you were right